config/environments/test.rb

```diff
@@ -18,6 +18,8 @@
   # system, or in some way before deploying your code.
   config.eager_load = ENV['CI'].present?
 
+  config.active_record.encryption.encrypt_fixtures = true
```

Comment:
Needed to encrypt attributes for fixtures that have it defined on the model attributes